ziti/edge/impl/conn.go

```diff
@@ -303,8 +304,8 @@ func (conn *edgeConn) establishServerCrypto(keypair *kx.KeyPair, peerKey []byte,
 func (conn *edgeConn) Listen(session *edge.Session, service *edge.Service, options *edge.ListenOptions) (edge.Listener, error) {
 	logger := pfxlog.Logger().
 		WithField("connId", conn.Id()).
-		WithField("service", service.Name).
-		WithField("session", session.Token)
+		WithField("serviceName", service.Name).
```

Comment:
should this have sessionId field as above?

Reply:
done